### ISSUE TITLE
Getting Started 'Building an app' displays a blank page

### ISSUE THREAD

**Issue by malikarun** [ionic-team/stencil#696]
**Stencil version:**
```
@stencil/core@0.7.15
```

**I'm submitting a:**
[X] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://stencil-worldwide.slack.com

**Current behavior:**
on macbook pro I run:
```
git clone https://github.com/ionic-team/stencil-app-starter my-app
cd my-app
git remote rm origin
npm install
npm start
```
The page displayed is blank, nothing seems to be injected into the DOM. 
Same steps above on iMac or linux machine work properly. But on a macbook pro (two separate machines) it doesn't work at all, it gives me a blank screen.
On a macbook, while running npm install, I got an error with fsevents@1.1.2, which I manually installed with `npm i fsevents`. The apps runs and opens the browser. It also reloads if I change code and save a file in the editor, but always shows a blank page.

Fails (shows the above behaviour) with :
-   Node versions (7.10.1, 8.11.1, 9.8.0, 9.11.0)
-   on a macbook pro (running macos High Sierra version 10.13.3)
-   and a macbook pro (running macos Sierra version 10.12.6)
-   and an iMac (running macos High Sierra version 10.13.2)

Works with:
-   an iMac (running macos Sierra version 10.12.6) with node 7.10.1
-   a linux machine (running archlinux) with node 9.8.0

**Expected behavior:**
I expected to see the browser to open  `localhost:3333` and show me the stencil starter template

**Steps to reproduce:**
```
git clone https://github.com/ionic-team/stencil-app-starter my-app
cd my-app
git remote rm origin
npm install
npm start
```
![screen shot 2018-04-05 at 2 34 03 pm](https://user-images.githubusercontent.com/3743409/38347413-aeda412a-38de-11e8-9e1e-4a2ab51900f8.png)
<img width="992" alt="screen shot 2018-04-05 at 2 34 38 pm" src="https://user-images.githubusercontent.com/3743409/38347414-b40fbaa8-38de-11e8-9c3b-d83cc2a5fa49.png">

**Comment by adamdbradley** [comment]
I updated the app to stencil 0.7.15 and unable to replicate the issue. @jgw96 would you be able to help debug this too?

**Comment by jgw96** [comment]
Looking into this now

**Comment by jgw96** [comment]
Hello all! I am also unable to reproduce this issue. This issue looks very similar to an issue we had in Stencil that has already been fixed and is in the 0.7.15 release. I would recommend double checking that you are definitely on @stencil/core@0.7.15. Because we are unable to repro, I am going to close this issue for now. Thanks for using Stencil!

**Comment by jgw96** [comment]
Also, I should note that I just updated the app starter to use 0.7.15 out of the box (:
